internal/querynode/collection_replica.go

```diff
@@ -296,6 +296,19 @@ func (colReplica *collectionReplica) getPartitionIDs(collectionID UniqueID) ([]U
 	return collection.partitionIDs, nil
 }
 
+func (colReplica *collectionReplica) getIndexedFieldIDByCollectionIDPrivate(collectionID UniqueID) ([]FieldID, error) {
+	fields, err := colReplica.getFieldsByCollectionIDPrivate(collectionID)
+	if err != nil {
+		return nil, err
+	}
+
+	fieldIDS := make([]FieldID, 0)
+	for _, field := range fields {
+		fieldIDS = append(fieldIDS, field.FieldID)
```

Comment:
should use func hasLoadIndexForIndexedField to check

Reply:
We check this outside.